mapzen-places-api/src/main/java/com/mapzen/places/api/internal/OnPlaceDetailsFetchedListener.java

```diff
@@ -0,0 +1,12 @@
+package com.mapzen.places.api.internal;
+
+/**
+ * Listener that is invoked by {@link PlaceDetailFetcher} when details for a place are retrieved.
+ */
+public interface OnPlaceDetailsFetchedListener {
```

Comment:
Even though this lives in the `internal` package we should still default to package private scope for interfaces and classes we don't want to be part of the public API when the Javadoc is generated.